build_docs.pl

```diff
@@ -371,6 +372,8 @@ sub check_kibana_links {
                 $path =~ s!\$\{KIBANA_DOCS\}!en/kibana/$branch/!;
                 $path =~ s!\$\{PLUGIN_DOCS\}!en/elasticsearch/plugins/$branch/!;
                 $path =~ s!\$\{FLEET_DOCS\}!en/fleet/$branch/!;
+                $path =~ s!\$\{APM_DOCS\}!en/apm/get-started/$branch/!;
```

Comment:
I don't think this will do what you want. It replaces `${APM_DOCS}` with `en/apm/get-started/$branch/`, then tries again to replace `${APM_DOCS}` with `en/apm/server/$branch/` which will never happen because it was replaced the first time. I think you'll need to use separate variables for the two different parts of the APM guide, unless you want to replace `${APM_DOCS}` with just `en/apm/` and then add either `get-started` or `server`, along with `${DOC_LINK_VERSION}` in the URLs you use in the `DocLinksService`.

And just to clarify, there's no reason you *need* to introduce a new variable in `DocLinksService`. You can always build links directly off of `${ELASTIC_WEBSITE_URL}`. 

Let me know if I can help.

Reply:
Thanks @gtback for the info. So if I delete one of the lines that I added and modify the other to:
`$path =~ s!\$\{APM_DOCS\}!en/apm/!`, should that work here? I'd like to check both `get-started` and `server`?

Reply:
Yep, that should work! And it looks like [this commit](https://github.com/elastic/kibana/pull/110992/commits/7368103c03585d35364b0b0bd7581665dc4edcf8) is already set up to handle it correctly.

Reply:
```suggestion
                $path =~ s!\$\{APM_DOCS\}!en/apm/!
```

Reply:
@gtback Okay, cool! Thanks for clarifying! I amended the PR accordingly.